Lesson8/recursion.cpp

```diff
@@ -0,0 +1,16 @@
+#include <iostream>
+#include "recursion.hpp"
+
+void printNaturalNumbersGoingDown(int count){
+    if(count > 0){
+        std::cout << count << " ";
+        printNaturalNumbersGoingDown(--count);
```

Comment:
count - 1; No need to change the value at this point

Reply:
fixed